src/actions/__tests__/cursorDown.ts

```diff
@@ -79,6 +85,33 @@ describe('normal view', () => {
 
     expectPathToEqual(stateNew, stateNew.cursor, ['a', 'm'])
   })
+
+  it('should navigate correctly in descending sorted list with formatted text', () => {
+    const text = `
+      - fruits
+        - =sort
+          - Alphabetical
+            - Desc
+        - apple
+        - orange
+        - banana
+        - pear
```

Comment:
It's not possible to have `=sort/Alphabetical/Desc` on a list of unsorted thoughts with normal use of the app. Activating Sort Descending would permanently rearrange the list.